lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -873,6 +893,7 @@
     description: "Dictionary of all the VM attributes. VM attributes can be found on your oVirt/RHV instance
                   at following url: http://ovirt.github.io/ovirt-engine-api-model/master/#types/vm."
     returned: On success if VM is found.
+        Additionally when you set ticket to true it returns ticket to remote-viewer and be careful with those private data.
```

Comment:
This should be part of `description`, after line 894

Reply:
Additionally when user sent ticket=true, this module will return also remote_vv_file parameter in vm dictionary, which contains remote-viewer compatible file to open virtual machine console. Please note that this file contains sensible information.